.eslintrc.js

```diff
@@ -63,59 +63,57 @@ module.exports = {
-        'no-var': 2,
-        'object-curly-spacing': [2, 'always'],
-        'object-curly-newline': [2, { consistent: true }],
-        'object-property-newline': [2, { allowAllPropertiesOnSameLine: true }],
-        'object-shorthand': [2, 'methods'],
-        'operator-linebreak': [0, 'before'],
-        'padded-blocks': [2, 'never'],
+        'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+        'keyword-spacing': ['error', { before: true, after: true }],
+        'linebreak-style': ['error', 'unix'], // no carriage returns
+        'max-len': ['error', { code: 120 }], // be friendly to laptops
+        'newline-after-var': 'error',
+        'require-atomic-updates': 'warn', // okay to warn?
+        'no-constant-condition': 'error',
+        'no-dupe-class-members': 'error',
+        'no-lonely-if': 'error',
+        'no-multiple-empty-lines': ['error', { max: 1, maxEOF: 1 }], // let my people push enter!
+        'no-prototype-builtins': 'off', // why would we need this off?
+        'no-underscore-dangle': 'error',
+        'no-var': 'error',
+        'object-curly-spacing': ['error', 'always'], // kinda wide 
+        'object-curly-newline': ['error', { consistent: true }],
+        'object-property-newline': ['error', { allowAllPropertiesOnSameLine: true }],
+        'object-shorthand': ['error', 'methods'],
+        'operator-linebreak': ['off', 'before'],
+        'padded-blocks': ['error', 'never'],
+        // adds a lot of blank lines 🤔
         'padding-line-between-statements': [
-            2,
+            'error',
             { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
             { blankLine: 'any', prev: ['const', 'let', 'var'], next: ['const', 'let', 'var'] },
             { blankLine: 'always', prev: '*', next: 'return' }
         ],
-        'prefer-arrow-callback': 2,
-        'prefer-const': 2,
-        'prefer-spread': 2,
-        'prefer-template': 2,
-        'quote-props': [2, 'as-needed'],
-        quotes: [2, 'single', 'avoid-escape'],
-        'semi-spacing': [2, { before: false, after: true }],
-        'space-before-blocks': 2,
+        'prefer-arrow-callback': 'error',
+        'prefer-const': 'error',
+        'prefer-spread': 'error',
+        'prefer-template': 'error',
+        'quote-props': ['error', 'as-needed'],
+        quotes: ['error', 'single', 'avoid-escape'],
+        'semi-spacing': ['error', { before: false, after: true }],
+        'space-before-blocks': 'error',
         'space-before-function-paren': [
-            2,
+            'error',
             {
                 anonymous: 'never',
                 named: 'never',
                 asyncArrow: 'always'
             }
         ],
-        'space-in-parens': 2,
-        'space-infix-ops': 2,
-        semi: [2, 'never'],
+        'space-in-parens': 'error',
+        'space-infix-ops': 'error',
+        semi: ['error', 'never'], // is it safe to do this? it's certainly prettier looking
```

Comment:
I'm definitely coming around to this one but want to ensure it's safe

Reply:
we talked through this one 🏅